examples/stable-diffusion/training/README.md

```diff
@@ -353,6 +353,12 @@ python ../../gaudi_spawn.py --world_size 8 --use_mpi train_dreambooth.py \
 
 ```
 similar command could be applied to loha, lokr, oft.
```

Comment:
To maintain consistency of this Readme document, please capitalize the first letter of each sentence in the dreambooth section (starting at Line 319) . Please also run make style.